utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -596,8 +596,11 @@ def create_argument_parser():
     option('--libicu', toggle_true('build_libicu'),
            help='build libicu')
 
-    option('--playgroundsupport', store_true('build_playgroundsupport'),
+    option('--playgroundsupport', toggle_true('build_playgroundsupport'),
            help='build PlaygroundSupport')
+    option(['--install-playgroundsupport'],
```

Comment:
Any reason for the list here?

Reply:
Mostly to group it, since the options seem not particularly organised.  I'm happy to move it elsewhere if you prefer.

Reply:
Oh, no I mean the list around `--install-playgroundsupport`.

Reply:
Oh, there is a super secret reason for that .... I would never cargo-cult a change *shifty eyes*.  I'm going to let the current run complete and then drop the list conversion (which, yes, is pointless)